src/schema/v1/me/index.ts

```diff
@@ -27,7 +27,7 @@ import FollowedArtists from "./followed_artists"
 import FollowedGenes from "./followed_genes"
 import FollowedShows from "./followed_shows"
 import FollowedFairs from "./followed_fairs"
-import Invoice from "./conversation/invoice"
+import { InvoiceType } from "./conversation/invoice"
```

Comment:
I think this should still import the default `Invoice` (which is now deprecated in that file).

Then, the `resolve` method in this file for the `me { invoice }` field can go away (that's done in `Invoice` as well).

Otherwise, you get the issue the Peril comment https://github.com/artsy/metaphysics/pull/2481#issuecomment-647589581 is alluding to.

Namely, a client still wanting to fetch an invoice by passing in those id arguments, will get a GraphQL validation error (instead of `null` for the invoice). I dunno if it matters too much but it might be a bit cleaner.

Reply:
@mzikherman You're exactly right, and the issue was also caught in the [test failure](https://circleci.com/gh/artsy/metaphysics/10011?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link). I've updated that in https://github.com/artsy/metaphysics/pull/2481/commits/111f7ed42f4a2c8f9605e5ca74e8ffbdb7e8d67f.

Re how older iOS clients are handling the invoice, I believe it's [checking the presence of the invoice](https://github.com/artsy/emission/pull/2091#issuecomment-587541873), so returning `null` should hopefully fall back gracefully.